desktop_version/CONTRIBUTORS.txt

```diff
@@ -1,7 +1,10 @@
 Contributors
 ------------
 
+(Ordered alphabetically by last name.)
```

Comment:
Actually I see now that prior to this commit contributors were ordered alphabetically by first name -- maybe that's better and we should keep it this way?